web/config/sync/user.role.moderator.yml

```diff
@@ -0,0 +1,14 @@
+uuid: d7b1ca0e-4271-4124-a560-9610e1a55679
+langcode: en
+status: true
+dependencies: {  }
+id: moderator
```

Comment:
Hmm, we might have to rename the `administrator` role to keep it consistent, if we decide on this, even though there are no administrators in the service.